keyboards/redragon/k552/v2/info.json

```diff
@@ -0,0 +1,300 @@
+            {"matrix": [2, 8], "flags": 4, "x": 89, "y": 29},
+            {"matrix": [2, 9], "flags": 4, "x": 99, "y": 29},
+            {"matrix": [2, 10], "flags": 4, "x": 109, "y": 29},
+            {"matrix": [2, 11], "flags": 4, "x": 120, "y": 29},
+            {"matrix": [2, 12], "flags": 4, "x": 130, "y": 29},
+            {"matrix": [2, 13], "flags": 4, "x": 143, "y": 29},
+
+            {"matrix": [2, 14], "flags": 4, "x": 159, "y": 29},
+            {"matrix": [2, 15], "flags": 4, "x": 169, "y": 29},
+            {"matrix": [2, 16], "flags": 4, "x": 180, "y": 29},
+
+            {"matrix": [3, 0], "flags": 4, "x": 4, "y": 41},
+            {"matrix": [3, 1], "flags": 4, "x": 18, "y": 41},
+            {"matrix": [3, 2], "flags": 4, "x": 29, "y": 41},
+            {"matrix": [3, 3], "flags": 4, "x": 39, "y": 41},
+            {"matrix": [3, 4], "flags": 4, "x": 49, "y": 41},
+            {"matrix": [3, 5], "flags": 4, "x": 60, "y": 41},
+            {"matrix": [3, 6], "flags": 4, "x": 70, "y": 41},
+            {"matrix": [3, 7], "flags": 4, "x": 81, "y": 41},
+            {"matrix": [3, 8], "flags": 4, "x": 91, "y": 41},
+            {"matrix": [3, 9], "flags": 4, "x": 102, "y": 41},
+            {"matrix": [3, 10], "flags": 4, "x": 112, "y": 41},
+            {"matrix": [3, 11], "flags": 4, "x": 122, "y": 41},
+            {"matrix": [3, 13], "flags": 4, "x": 139, "y": 41},
+
+            {"matrix": [4, 0], "flags": 4, "x": 7, "y": 52},
+            {"matrix": [4, 2], "flags": 4, "x": 23, "y": 52},
+            {"matrix": [4, 3], "flags": 4, "x": 34, "y": 52},
+            {"matrix": [4, 4], "flags": 4, "x": 44, "y": 52},
+            {"matrix": [4, 5], "flags": 4, "x": 55, "y": 52},
+            {"matrix": [4, 6], "flags": 4, "x": 65, "y": 52},
+            {"matrix": [4, 7], "flags": 4, "x": 76, "y": 52},
+            {"matrix": [4, 8], "flags": 4, "x": 86, "y": 52},
+            {"matrix": [4, 9], "flags": 4, "x": 96, "y": 52},
+            {"matrix": [4, 10], "flags": 4, "x": 107, "y": 52},
+            {"matrix": [4, 11], "flags": 4, "x": 117, "y": 52},
+            {"matrix": [4, 13], "flags": 4, "x": 137, "y": 52},
+
+            {"matrix": [4, 15], "flags": 4, "x": 169, "y": 52},
+
+            {"matrix": [5, 0], "flags": 4, "x": 1, "y": 64},
+            {"matrix": [5, 1], "flags": 4, "x": 14, "y": 64},
+            {"matrix": [5, 2], "flags": 4, "x": 27, "y": 64},
+            {"matrix": [5, 3], "flags": 4, "x": 66, "y": 64},
+            {"matrix": [5, 4], "flags": 4, "x": 105, "y": 64},
+            {"matrix": [5, 5], "flags": 4, "x": 119, "y": 64},
+            {"matrix": [5, 6], "flags": 4, "x": 132, "y": 64},
+            {"matrix": [5, 8], "flags": 4, "x": 145, "y": 64},
+
+            {"matrix": [5, 14], "flags": 4, "x": 159, "y": 64},
+            {"matrix": [5, 15], "flags": 4, "x": 169, "y": 64},
+            {"matrix": [5, 16], "flags": 4, "x": 180, "y": 64},
```

Comment:
coordinates need normalization. x_max= 224, y_max=64. Otherwise effects will not scale properly and you will effectively have a chopped up fullsize board instead of a TKL

Reply:
Should be fine in the latest iteration :)
At least it looks a LOT more aligned on my board now! :)